### DISCUSSION TITLE
Another 0Day's Coming to Add

### DISCUSSION THREAD

**Discussion by RashidKhanPathan** [RashidKhanPathan/Security-Research#3] [Q&A] [answered]
Thanks for Adding CVE-2022-37059 0Day Research on Mine Resume, also i will Provide Upcoming Developed 0Day's Here, so Please Keep Stay Here, we will discuss Here

**Comment by iHexCoder** [comment, marked as answer]
Sounds Great, Let me Know on Email First after Getting CVE's for Those 0Day's
